### PR TITLE
ticket 6 pull request

### PR DESCRIPTION
**Pull request by KarlLuther**
Hello

Everything seems to work fine apart from the errors: I am not sure whether I am handling them correctly 

Thank you
Ivan

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+          expect(comment).toHaveProperty("body", expect.any(String));
+          expect(comment).toHaveProperty("author", expect.any(String));
+          expect(comment).toHaveProperty("comment_id", expect.any(Number));
+          expect(comment).toHaveProperty("votes", expect.any(Number));
+          expect(comment).toHaveProperty("created_at", expect.any(String));
+        }
+      });
+  });
+  it("if comments from review with id = three are requested should respond with an array of comment objects with corresponding reviews_id and other properties", () => {
+    return request(app)
```
Comment (pippaamy) [inline __tests__/app.test.js:122]: Can we add a status 200 test that if it has a valid id but has no comments it responds with an empty array of comments.

**Review by pippaamy — CHANGES_REQUESTED** [review]
Good work Ivan, I would just add another test that I've popped in there. You may have to add more functionality to help that pass!

**Review by Hard-Luck — APPROVED** [review]
good to go!
